java/org/contikios/cooja/plugins/TimeLine.java

```diff
@@ -589,11 +591,21 @@ private static double zoomLevelToDivisor(int zoomLevel) {
     return ZOOM_LEVELS[zoomLevel];
   }
 
-  private void zoomFinish (final double zoomDivisor,
+  private void zoomFinish (double zoomDivisor,
                            final long focusTime,
-                           final double focusCenter) {
-    currentPixelDivisor = zoomDivisor;
+                           final double focusCenter) 
```

Comment:
Keep the opening brace where it was (this is how almost all code in Cooja is formatted).